Modules/Filtering/ImageFeature/test/itkDerivativeImageFilterTest.cxx

```diff
@@ -62,6 +62,20 @@ itkDerivativeImageFilterTest(int argc, char * argv[])
   filter->SetOrder(std::stoi(argv[3]));
   filter->SetDirection(std::stoi(argv[4]));
 
+  auto useImageSpacing = true;
```

Comment:
This can be constexpr, and the else block at line 71-74 is never used.

Reply:
Sorry Hans. Looks like you did not pick the last commit. This was intentional for commit 84808d7, but commit c19e8fa  improves the test by adding a parameter that bears the information for this variable. Note that the changes in the commits respond to the purpose of the commits. Hope this explains it.

Reply:
OK.